neo/d3xp/script/Script_Interpreter.cpp

```diff
@@ -176,135 +178,166 @@ void idInterpreter::Reset( void ) {
 	doneProcessing	= true;
 }
 
-/*
-================
-idInterpreter::GetRegisterValue
-
-Returns a string representation of the value of the register.  This is
-used primarily for the debugger and debugging
-
-//FIXME:  This is pretty much wrong.  won't access data in most situations.
-================
-*/
-bool idInterpreter::GetRegisterValue( const char *name, idStr &out, int scopeDepth ) {
+bool idInterpreter::GetRegisterValue(const char* name, idStr& out, int scopeDepth) {
 	varEval_t		reg;
-	idVarDef		*d;
-	char			funcObject[ 1024 ];
-	char			*funcName;
-	const idVarDef	*scope;
-	const idTypeDef	*field;
-	const idScriptObject *obj;
-	const function_t *func;
+	idVarDef* d;
+	char			funcObject[1024];
+	char* funcName;
+	const idVarDef* scope = NULL;
+	const idVarDef* scopeObj;
+	const idTypeDef* field;
+	const function_t* func;
 
 	out.Empty();
 
-	if ( scopeDepth == -1 ) {
+	if (scopeDepth == -1) {
 		scopeDepth = callStackDepth;
 	}
 
-	if ( scopeDepth == callStackDepth ) {
+	if (scopeDepth == callStackDepth) {
```

Comment:
a lot more unnecessary whitespace changes around here

for the future, could you configure VS to disable auto-formatting (or to only do it for new lines you add, not when modifying existing code)?

Reply:
damnit! i did figure this out man i fixed this! very anoying, i would prefer we use an cross IDE solution so that is consistent 
Like i asked last time? clang format for example?

Reply:
formatting tools are not an option - they will format *all* code and thus will fuck up `git blame` for all code.

ideally editors should only (re)format lines you've actually changed and not surrounding ones or whatever has been happening here